apps/code-shaper/src/index.ts

```diff
@@ -1,41 +1,42 @@
-import inquirer from 'inquirer';
 // @ts-ignore
-import inquirerDirectory from 'inquirer-directory';
+import inquirer from 'inquirer';
 import yargs from 'yargs/yargs';
-import { generateGenerator } from './generator-generator/generateGenerator';
-import { generateRepo } from './repo-generator/generateRepo';
-
-const generators = [
-  { name: 'Repository generator', value: 'repo' },
-  { name: 'Generator generator', value: 'generator' },
-];
+import {getPlugin, Plugin, getAvailablePlugins} from '@code-shaper/shaper-utils';
 
 async function main() {
   const argv = await yargs().parse(process.argv.slice(2));
 
-  const questions = [
-    {
-      type: 'list',
-      name: 'generator',
-      message: 'Which generator would you like to run?',
-      choices: generators,
-    },
-  ];
-
-  inquirer.registerPrompt('directory', inquirerDirectory);
-  const context = await inquirer.prompt(questions, argv);
-
-  // Execute generator
-  switch (context.generator) {
-    case 'repo':
-      await generateRepo(context);
-      break;
-    case 'generator':
-      await generateGenerator(context);
-      break;
-    default:
-      console.error(`Unknown generator: ${context.generator}`);
-      break;
+  const nameArg = argv._[0] as string;
```

Comment:
We can't take over `shaper` because that's our bootstrapping cli. Imagine that you had no repo whatsoever. How would you even run shaper? There is no `node_modules` or `pluginDir` at that point. So this code should be moved over to myshaper that is generated by shaper.

So the workflow should be like this:

1. Run shaper to create a blank repo. This repo will have a myshaper with the automatic plugin detection.
2. npm install plugins like react, nextjs in this new repo.
3. Now running myshaper in this repo will start detecting plugins.